src/main/webapp/js/instructor.es6

```diff
@@ -369,6 +369,25 @@ function attachEventToDeleteStudentLink() {
     });
 }
 
+function remindButtonAjax(urlLink) {
```

Comment:
@chowyb `oncontextmenu` could be used to disable right right click on these type of buttons.
Same is the case with publish/unpublish results and there might be some more!